keyboards/handwired/yamsek/atmega32u4/rules.mk

```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = caterina
+
+RGBLIGHT_ENABLE = no
+RGBLIGHT_DRIVER = WS2812
+
+POINTING_DEVICE_ENABLE = no
+POINTING_DEVICE_DRIVER = pimoroni_trackball
+
+## saving space to squeeze in VIAL support
+# (by explicitly disabeling features, some of which might be auto-enabled by the build scripts)
+AUTO_SHIFT_ENABLE = no
+COMBO_ENABLE = no
+DEBUG_ENABLE = no
+#  KEY_OVERRIDE_ENABLE            = yes              # file
+#  OLED_DRIVER                    = SSD1306          # file
+#  RAW_ENABLE                     = yes              # file
+TAP_DANCE_ENABLE = no
+
```

Comment:
The firmware does not change with/without these things
```shell
~
❯ sha512sum handwired_yamsek_atmega32u4_default.hex
a4e266bd5f4807368f63618b61d7067d4cff487ca82d68d76af560807f330a49e07d907d479301ecd93e4365b08950b64777cdfde86905394824fccd2d30a813  handwired_yamsek_atmega32u4_default.hex

~
❯ sha512sum handwired_yamsek_atmega32u4_default_no_feature.hex
a4e266bd5f4807368f63618b61d7067d4cff487ca82d68d76af560807f330a49e07d907d479301ecd93e4365b08950b64777cdfde86905394824fccd2d30a813  handwired_yamsek_atmega32u4_default_no_feature.hex
```
(MCU data belongs in info.json file, which is missing)
```suggestion
# This file intentionally left blank
```